ads_plugin.py

```diff
@@ -432,11 +433,11 @@ def on_save_segmentation_button(self, event):
         myelin_array = myelin_array * params.intensity['binary']
         axon_array = axon_array * params.intensity['binary']
 
-        # Save the arrays as PNG files
+
         myelin_and_axon_array = (myelin_array // 2 + axon_array).astype(np.uint8)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg.png", img=myelin_and_axon_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-myelin.png", img=myelin_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-axon.png", img=axon_array)
+        ads_utils.imwrite(filename=save_dir + "/" + self.png_image_name[0][:-4] + axonmyelin_suffix.name, img=myelin_and_axon_array)
+        ads_utils.imwrite(filename=save_dir + "/" + self.png_image_name[0][:-4] + myelin_suffix.name, img=myelin_array)
+        ads_utils.imwrite(filename=save_dir +"/" + self.png_image_name[0][:-4] + axon_suffix.name, img=axon_array)
```

Comment:
@Stoyan-I-A this is related to your comment (452). I was thinking about a use case when the user loads let's say n images and the user wishes to save all the segmentations. In that case shouldn't we use `self.png_image_name` (which contains all the image names) to save all the segmentations ?

Reply:
We could do that as long as the name of the saved masks correspond to the right image.

Reply:
For now, I've used overlays name to save the segmentations because the [on_save_segmentation_button](https://github.com/neuropoly/axondeepseg/blob/master/ads_plugin.py) doesn't allow to save all the segmentations ( see [here](https://github.com/neuropoly/axondeepseg/issues/445#issuecomment-768673429)) and this would require some changes in the plugin. So, to keep things simple and fix naming convention through this PR (I've made minimal changes).